src/index.html

```diff
@@ -11,6 +11,44 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form 
+   action="google.com" 
+   method="post"
+   class="form--big"
+   data-qa="big">
```

Comment:
[Checklist](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md)

Reply:
not fixed